buildSrc/src/main/resources/pom.xml

```diff
@@ -12,6 +12,16 @@
         </repository>
     </repositories>
     <dependencies>
+        <dependency>
+            <groupId>org.graalvm.polyglot</groupId>
+            <artifactId>polyglot</artifactId>
+            <version>23.1.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.graalvm.polyglot</groupId>
+            <artifactId>js</artifactId>
+            <version>23.1.0</version>
```

Comment:
This needs a <type>pom</type>

Reply:
we don't use this pom.xml as a typical pom.xml but as a way define versions coordinates.

Reply:
Sorry, I cannot follow. Afaik with Maven, you need to specify that a dependency is of type pom if the target dependency is a pom type dependency on Maven central. Otherwise it fails badly.